api-operator/pkg/registry/registry.go

```diff
@@ -55,25 +55,30 @@ type Config struct {
 	Env              []corev1.EnvVar               // Environment variables to be set in the pod that runs Kaniko Job
 	Args             []string                      // Args to be passed to the Kaniko Job
 	ImagePullSecrets []corev1.LocalObjectReference // Secrets for the pod which runs the final micro-gateway setup
-	IsImageExist     func(config *Config, auth utils.RegAuth, image string,
+	IsImageExist     func(config *Config, auth utils.RegAuth, repo string, img string,
 		tag string) (bool, error) // Function to check the already existence of the image
 }
 
 // registry details
-var dockerImage Image
+//var dockerImage Image
 var registryConfigs = map[Type]func(repoName string, imgName string, tag string) *Config{}
 
 // SetRegistry sets the registry type, repository and image
 func SetRegistry(client *client.Client, namespace string, img Image) error {
 	logger.Info("Setting registry type", "image", img)
-	dockerImage = img
+	//	dockerImage = img
 
-	return copyConfigVolumes(client, namespace)
+	return copyConfigVolumes(client, namespace, img)
 }
 
 // GetConfig returns the registry config
-func GetConfig() *Config {
-	return registryConfigs[dockerImage.RegistryType](dockerImage.RepositoryName, dockerImage.Name, dockerImage.Tag)
+//func GetConfig() *Config {
```

Comment:
Shall we remove these commented out code? There's commented out code in several places in this file.

Reply:
Sorry this is not the latest changes I've made. I'll update them

Reply:
@RakhithaRR I've made the changes, there should be no more commented code anymore